packages/fxa-payments-server/server/lib/amplitude.test.js

```diff
@@ -0,0 +1,26 @@
+const amplitude = require('./amplitude.js');
```

Comment:
Don't need the `.js` after `amplitude` here.

Reply:
The typescript module loader complained when the .js was omitted :-\

Reply:
Ah, it was Jest that couldn't find the modules without the .js extension. Just ran into that error again on CI.